### ISSUE TITLE
dateparser.parse('02/2013', ['%m/%Y']) gives wrong datetime

### ISSUE THREAD

**Issue by mcarans** [scrapinghub/dateparser#607]
dateparser.parse('02/2013', ['%m/%Y']) gives datetime(2013, 2, 28, 0, 0) instead of datetime(2013, 2, 1, 0, 0)

datetime.strptime('02/2013', '%m/%Y') gives datetime(2013, 2, 1, 0, 0)

**Comment by noviluni** [comment]
In this moments `dateparser` returns the last day of the month for this kind of formats. I think the reason behind that decission is that when finding only a month and a year we usually refer to the last day of that month (i.e. expiration date).

HOWEVER that is a bug/inconsistency, because there is a setting `PREFER_DAY_OF_MONTH` that should be followed, as when we just write `dateparser.parse('February 2013')`.

Examples:

```
# It works

dateparser.parse('February 2013', settings={'PREFER_DAY_OF_MONTH': 'current'})                                                                                                                          
>>> datetime.datetime(2013, 2, 17, 0, 0)

dateparser.parse('February 2013', settings={'PREFER_DAY_OF_MONTH': 'first'})                                                                                                                            
>>> datetime.datetime(2013, 2, 1, 0, 0)

dateparser.parse('February 2013', settings={'PREFER_DAY_OF_MONTH': 'last'})                                                                                                                             
>>> datetime.datetime(2013, 2, 28, 0, 0)

# It doesn't work

dateparser.parse('02/2013', settings={'PREFER_DAY_OF_MONTH': 'current'})                                                                                                                          
>>> datetime.datetime(2013, 2, 28, 0, 0)

dateparser.parse('02/2013', settings={'PREFER_DAY_OF_MONTH': 'first'})                                                                                                                            
>>> datetime.datetime(2013, 2, 28, 0, 0)

dateparser.parse('02/2013', settings={'PREFER_DAY_OF_MONTH': 'last'})                                                                                                                             
>>> datetime.datetime(2013, 2, 28, 0, 0)


```

I added a PR with a patch.

**Comment by jbcooley7** [comment]
I've just run across this issue too. Supposing the PR gets final approval and is merged, is there a projected release date at which this fix will go out? I saw the most recent release was September 2019, but prior releases seemed to be on a nearly yearly cadence around the first months of the year.

**Comment by noviluni** [comment]
Hi @jbcooley7 this has been fixed in the last version `0.7.4` :tada: 

We will try to start doing more frequent releases. 

Cheers!
